docs/tutorials/index.md

```diff
@@ -5,33 +5,33 @@ Learn how to use Taipy and improve your skills with the following tutorials.
 ## Coding tutorials
 | Id  |            Title             |                                    description                                    |  keywords  |                                                        link to the markdown page | 
 |:----|:----------------------------:|:---------------------------------------------------------------------------------:|:----------:|---------------------------------------------------------------------------------:|
-| 1   |      Understanding GUI       | Understand basic knowledge of Taipy GUI creating of a multi-page NLP application. |  **TODO**  |                     [gui tuto](complete_applications/understanding_gui/index.md) |
-| 2   | Scenario management Overview |             Learn the main features of the Taipy scenario management              |  **TODO**  | [scenario mgt tuto](complete_applications/scenario_management_overview/index.md) |
-| 3   | First Realistic application  |              Build a complete application from back-end to front-end              |  **TODO**  | [Complete application tuto](complete_applications/complete_application/index.md) |
+| 1   |      Understanding GUI       | Gain a fundamental understanding of creating a multi-page NLP application using Taipy front-end. |  Application front-end building  |                     [gui tuto](complete_applications/understanding_gui/index.md) |
```

Comment:
I am not sure about this title. Maybe "Gain a fundamental understanding of Taipy front-end by creating a multi-page NLP application"?